source/guide_pleroma.rst

```diff
@@ -42,13 +42,21 @@ Your URL needs to be setup:
 Setup PostgreSQL
 ----------------
 
-You should've running a :lab:`PostgreSQL <guide_postgresql>` in the version ``9.6`` or above configured with ``--with-uuid=e2fs``
+We’re using :manual:`PostgreSQL <database-postgresql>` in the version 13. Please refer to the :lab:`PostgreSQL <guide_postgresql>` guide to setup your instance.
 
 ::
 
- [isabell@stardust ~]$ ./configure --prefix=$HOME/opt/postgresql/ --with-python PYTHON=/usr/bin/python2 --without-readline --with-uuid=e2fs
+ [isabell@stardust ~]$ uberspace tools version show postgresql
+ Using 'postgresql' version: 13
  [isabell@stardust ~]$
 
+Please make sure you set ``listen_addresses`` in the config.
+
+.. code-block:: console
+ :emphasize-lines: 1
+
+ listen_addresses = '*'         # what IP address(es) to listen on;
```

Comment:
isn't [the default value localhost](https://www.postgresql.org/docs/13/runtime-config-connection.html#GUC-LISTEN-ADDRESSES) enough here since we only care about access within the uberspace?

Reply:
I tested it and it worked with ``#listen_addresses = 'localhost'`` too. PSQL still listen on localhost with the default config.
Thought psql dont listen on any interface when this line is comment out by a ``#``.